TF_Application/Assets/TouchFree/Tooling/Scripts/Connection/MessageReceiver.cs

```diff
@@ -105,6 +105,7 @@ void HandleResponse(WebSocketResponse _response)
                 if (callback.Key == _response.requestID)
                 {
                     callback.Value.callback.Invoke(_response);
+                    Debug.LogWarning(_response.message);
```

Comment:
Does this need to be a warning? it could be a good response?

Reply:
This change is already in the main tooling, I've just copied this to the Overlay application. I can swap this in both places if you think "Log" would be more appropriate?